coding-concepts.js

```diff
@@ -7,62 +7,67 @@
 
 // --------------------1) What will this log?
 
-const cohort = "Echo 2023"
+// const cohort = "Echo 2023"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["E" "c" "h" "o" "2" "0" "2" "3"]
+// b) Verify and explain: It did log what I thought, but I did forget the space in the middle and the commas. Splits a string into an array, but every single entry(including spaces, seperated by commas).
 
 // --------------------2) What will this log?
 
-const greeter = (name) => {
-  `Hello, ${name}!`
-}
+// const greeter = (name) => {
+//   `Hello, ${name}!`
+// }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: An error of some type? Maybe undefined? since I don't think you can add a string into the function invoking brackets like that. It should just be added into the string interpolation instead.
+// b) Verify and explain: It wasundefined, not exactly a proper error, but still can be fixed with the string interpolated message instead.
```

Comment:
It gives undefined because there is no return in the function. If you add a return, everything else is fine.